lib/Conversion/TosaToXTenNN.cpp

```diff
@@ -169,28 +204,10 @@ class FoldMulsToQDQOps : public OpRewritePattern<tosa::MulOp> {
           "i/o shape cannot change when multiplying due to broadcasting.");
     }
 
-    // Attempt to convert the scale factors to a log2 base. And ensure that both
-    // are equal. The quantization factor being the negation of the
```

Comment:
Why do the shift values no longer need to be equal?

Reply:
Good catch! I accidentally remove it. Will add back the check.

Reply:
Ok, then I have a follow up: If they need to be equal, don't we also want a sorting that moves the "correct" power of two to the RHS in case we have a multiplication of two constants and they are both powers of two?

Reply:
Oh yes that sounds like a much better way to handle this 🙂